app/src/interfaces/map/style.ts

```diff
@@ -0,0 +1,207 @@
+export default [
+	{
+		id: 'directus-polygon-fill-inactive',
+		type: 'fill',
+		filter: ['all', ['==', 'active', 'false'], ['==', '$type', 'Polygon'], ['!=', 'mode', 'static']],
+		paint: {
+			'fill-color': '#3bb2d0',
+			'fill-outline-color': '#3bb2d0',
+			'fill-opacity': 0.1,
```

Comment:
Can we use CSS Vars here? Makes sure we have dark-mode support covered too.

Reply:
Nope, these values are used in maplibre WebGL context. We would need watchers for css variables (does that exists ?) that dynamically recreate a json style whenever these would change. I'm not even sure it makes sense, as the basemap is selected by the user (what is a dark mode for an orthographic map ?).

A solution would be to have a "Default" option in the basemap select, that automatically switch between two predefined (in the admin setting also ?) theme, one for dark and one for light. If we add the ESRI OSM vector basemaps, these could be it.

Reply:
We could turn this into a function that will run everytime libre is initialized, that way we can pull the css var the moment it's being rendered (doesn't have to be fully live/watched)

> I'm not even sure it makes sense, as the basemap is selected by the user (what is a dark mode for an orthographic map ?).

True 👍🏻 

> A solution would be to have a "Default" option in the basemap select, that automatically switch between two predefined (in the admin setting also ?) theme, one for dark and one for light. If we add the ESRI OSM vector basemaps as default collection in Directus, these could be it, otherwise the admin would have to select them.

It would be great if we could figure out the theming on the default settings of the map. Just a little worried to have a giant white square in full screen dark mode in an otherwise nice and dim environment 🙂